site/docs/writing_stardoc.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title: Writing Documentation
+---
+
+## Rule Documentation
+
+When generating documentation, Stardoc parses the `.bzl` file to extract the
+inline documentation as well as evaluates the Starlark code to determine the
+types for rule attributes.
+
+Private rule attributes (i.e. those whose names begin with `_`) will not
```

Comment:
Replace "i.e." with "that is" or equivalent. Google-wide practice is to avoid i.e. and e.g. since non-English speakers may not know what they stand for.